src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+class Task {
+    private final boolean status;
+    private final String taskDescription;
+
+    Task(boolean s, String td) {
```

Comment:
Maybe you want to replace 's' with a meaningful word? Based on java coding standard, for boolean, the declared variable will start with "is", e.g: isDone